<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionWindow</class>
 <widget class="QMainWindow" name="TransactionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 237, 215);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,182,46);</string>
    </property>
    <property name="text">
     <string>Add Transactions</string>
    </property>
   </widget>
   <widget class="QLabel" name="balancelabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,182,46);</string>
    </property>
    <property name="text">
     <string>Your Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="balancemoney">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,182,46);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,182,46);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="footer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>661</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="homeicon">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>585</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Champagne &amp; Limousines</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);
background-image : url(img/homenotfilled.png)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="addicon">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>585</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Champagne &amp; Limousines</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);
background-image : url(img/addfilled.png)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="homebutton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>569</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);</string>
    </property>
    <property name="text">
     <string>


Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addbutton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>569</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);</string>
    </property>
    <property name="text">
     <string>



Add Transactions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statementlabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>569</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);</string>
    </property>
    <property name="text">
     <string>


Statements</string>
    </property>
   </widget>
   <widget class="QLabel" name="statementicon">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>585</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Champagne &amp; Limousines</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200,224,218);
background-image : url(img/notes.png)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="savebutton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);
border-width: 2px;
border-radius: 20px;
padding: 4px;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="note">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226,226,226);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="note_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Note :</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 237, 215);
selection-background-color: rgb(200, 224, 218);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Income</string>
     </attribute>
     <widget class="QLabel" name="body">
      <property name="geometry">
       <rect>
        <x>-130</x>
        <y>-30</y>
        <width>661</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gagalin</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
background-color: rgb(248, 237, 215);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="currlabel2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string>Rp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="incline">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 234, 234);
border-color: rgb(234, 234, 234);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmbutton1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Expenditure</string>
     </attribute>
     <widget class="QLabel" name="currlabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Rp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="expline">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 234, 234);
border-color: rgb(234, 234, 234);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmbutton2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="currlabel3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gagalin</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 182, 46);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Rp</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(200, 224, 218);</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2021</year>
      <month>10</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <zorder>footer</zorder>
   <zorder>addbutton</zorder>
   <zorder>header</zorder>
   <zorder>welcomelabel</zorder>
   <zorder>balancelabel</zorder>
   <zorder>balancemoney</zorder>
   <zorder>homebutton</zorder>
   <zorder>homeicon</zorder>
   <zorder>addicon</zorder>
   <zorder>statementlabel</zorder>
   <zorder>statementicon</zorder>
   <zorder>line</zorder>
   <zorder>savebutton</zorder>
   <zorder>note</zorder>
   <zorder>line_2</zorder>
   <zorder>note_label</zorder>
   <zorder>tabWidget</zorder>
   <zorder>currlabel3</zorder>
   <zorder>date</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
